CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [4.1.0] - 2020-08-19
+
+### Summary
```

Comment:
We usually had a short summary of every change here, but I don't have strong feelings about it.

Reply:
Updated!